src/flutter/shell/platform/linux_embedded/window/linuxes_window_drm.cc

```diff
@@ -97,7 +95,12 @@ bool LinuxesWindowDrm::DispatchEvent() {
 }
 
 bool LinuxesWindowDrm::CreateRenderSurface(int32_t width, int32_t height) {
-  native_window_ = std::make_unique<NativeWindowDrm>(kDrmDeviceFilename);
+  auto device_filename = getenv("DRM_DEVICE_FILENAME");
```

Comment:
Why do you use C library? You should use C++ standard library like `std::getenv` instead of C library.